appinventor/appengine/src/com/google/appinventor/client/style/neo/neo.css

```diff
@@ -78,11 +78,10 @@ they get cut off in Windows.
     padding-left: unset;
   }
 
-  .ode-Box {
```

Comment:
I feel like there are other components that rely on the definition of `.ode-Box`. Are we sure this doesn't break anything? A safer approach would be to introduce a new class rather than renaming an existing class.

Reply:
> I feel like there are other components that rely on the definition of `.ode-Box`. Are we sure this doesn't break anything? A safer approach would be to introduce a new class rather than renaming an existing class.

The class exists in the java code, but there's no css definition for it in ya.css. I added that height specification, and it turned out it wasn't good for the Assets and Component Explorer boxes stacked on top of each other. 

Neo.css is loaded on top of ya.css anyway, so if there were a definition of the class there, it would apply to Neo.

Reply:
ode-Box is used in Box.java, which is the parent of many containers...

Reply:
We may want to adjust the definition to `.ode-Box, .ode-Box-height-100 {...}` to not break any existing semantics.

Reply:
> We may want to adjust the definition to `.ode-Box, .ode-Box-height-100 {...}` to not break any existing semantics.

That makes sense. I'll do that.

Reply:
Ok, after further investigation I understand what you're saying. The .ode-Box CSS rule was added as a previous fix to another new UI issue, so this is a "fix" for that fix.

Reply:
For some reason I thought in my head that the .ode-Box rule was older than the last few months.